models/src/main/thrift/content/v1.thrift

```diff
@@ -760,6 +759,8 @@ struct BlockElement {
 
     20: optional ContentAtomElementFields contentAtomTypeData
 
+    21: optional EmbedTracking tracking
```

Comment:
Think it would be helpful to include a comment as to why this field is optional.